db/migrate/20240109223117_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.integer :user_id
+      t.integer :post_id
```

Comment:
👍 Good job on creating the migration file for the `likes` table! However, there's a crucial element missing in the current setup.

- [ ] Upon reviewing the database diagram, it's evident that the `likes` table should have a one-to-many relationship with the `users` table and another one-to-many relationship with the `posts` table. This association is essential for linking likes to their respective posts and users. To address this, please follow the [official documentation](https://guides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references) on creating associations using references. Update your migration file accordingly to establish the relationship between the `likes` to `users` and `posts` tables using `user_id` and `post_id`. After making the necessary changes, run the migration files to ensure that the code works successfully.


Keep up the good work! If you have any questions or need further guidance, feel free to ask. 🚀👍

Reply:
Thanks for the review @shahadat3669 , but as you know, when we put the belongs_to in migration, it's just add  tablename_id as a new column, and the proper association is set in model, so that why I decided to create all the foreign column manually and set the association in the column ... is it bad to do that ??

Reply:
@francksefu, Good Question. 👍 

- In the current milestone, your focus is primarily on the migration file. I've observed that you attempted to establish relationships using models, which is commendable. However, it's crucial to note that at the database level, the relationship hasn't been created yet. Given that this milestone emphasizes the database layer rather than the model layer, it's essential to ensure the proper relationship is established at the database level. Please address this in your current work.

- Additionally, I noticed that you've created an index for the comments table. It's important to remember that we typically create indexes for foreign keys. Therefore, it's imperative to establish the relationship at the database level. 
- I'd like to highlight that if you create a foreign key in the migration table using the references keyword, it will automatically generate the corresponding index for the foreign key. You can refer to the Rails documentation [here](https://guides.rubyonrails.org/active_record_migrations.html#references) for more details on this feature.

Reply:
Thanks for give me light